### PR TITLE
Wizard.js - Allow confirmAction prop to override confirm button text

### PR DESCRIPTION
**Pull request by kruai**
Add an optional override to the "Confirm" button's text.
This was created so that a Wizard created in Remediations could have the button say "Save" instead.

### CODE REVIEW

File: src/PresentationalComponents/Wizard/Wizard.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ class Wizard extends Component {
                     variant="primary"
                     isDisabled={ !isValidated }
                     onClick={ () => this.handleOnClose(true) }>
-                        Confirm
+                    { this.props.confirmAction }
```
Comment (karelhala) [inline src/PresentationalComponents/Wizard/Wizard.js]: Please do not reference confirm action trough `this.props` but rather add `confirmAction` to https://github.com/RedHatInsights/insights-frontend-components/blob/72aa2d227bc92230f82397a7b90ecd2af51f84d6/src/PresentationalComponents/Wizard/Wizard.js#L45

**Comment by karelhala** [comment]
Please add tests to this functionality and run `npm t -- -u` to update snapshots as well.

File: src/PresentationalComponents/Wizard/Wizard.js (hunk 1 above)
Comment (kruai) [reply src/PresentationalComponents/Wizard/Wizard.js]: Will do. Is this just for readability, or is there a performance difference as well?

Comment (karelhala) [reply src/PresentationalComponents/Wizard/Wizard.js]: Mostly for readability. There might be a slight performance difference, but that won't be noticable either way.

**Pull request merged**